### PR TITLE
Fix reinstall-missing-ap pand remove indexer from status

### PR DESCRIPTION
**Pull request by poupotte**

### CODE REVIEW

File: lib/stack_application.coffee [hunk 1]
```diff
@@ -199,42 +198,31 @@ module.exports.updateAll = (callback) ->
...
+        path = "#{appsPath}/#{name}/#{name}/cozy-#{name}/package.json"
+    if fs.existsSync path
+        data = fs.readFileSync path, 'utf8'
+        data = JSON.parse data
+        callback data.version
     else
-        if name is "controller"
-            path = "/usr/local/lib/node_modules/cozy-controller/package.json"
+        if name is 'controller'
+            path = "/usr/lib/node_modules/cozy-controller/package.json"
```
Comment (nono) [inline lib/stack_application.coffee]: @poupotte I have no `/usr/lib/node_modules/cozy-controller/package.json` on my cozy. But I have a one in `/usr/local`. Are you sure about this change?

Comment (poupotte) [inline lib/stack_application.coffee]: Actually, I just remove `if name is 'indexer' ... else`, and remove an indentation. Git diff isn't clear.

The two paths can used : 
 Line 203, we test path `/usr/local/lib/node_modules/cozy-controller/package.json`, if this path doesn't exist (line 211), we try with path `/usr/lib/node_modules/cozy-controller/package.json`

Comment (nono) [inline lib/stack_application.coffee]: OK, my bad. I was lost in the diff.

**Comment by nono** [comment]
Thanks @poupotte !

**Pull request merged**